static/src/javascripts/projects/common/modules/experiments/tests/adblocking-response.js

```diff
@@ -0,0 +1,158 @@
+define([
+    'common/utils/$',
+    'common/utils/config',
+    'common/utils/detect',
+    'common/utils/storage',
+    'lodash/collections/contains',
+    'common/modules/commercial/user-features',
+    'common/modules/commercial/survey/survey-adblock',
+    'common/utils/cookies'
+], function (
+    $,
+    config,
+    detect,
+    storage,
+    contains,
+    userFeatures,
+    SurveyAdBlock,
+    cookies
+) {
+    return function () {
+        this.id = 'AdBlockingResponse';
+        this.start = '2016-08-17';
+        this.expiry = '2016-10-18';
+        this.author = 'Justin Pinner';
+        this.description = 'Adblocking response ZERO PERCENT TEST WITH 10 MINUTES GRACE';
+        this.audience = 0;
+        this.audienceOffset = 0;
+        this.successMeasure = 'Adblocking users will either pay, prove they have a subscription or turn off their adblocker.';
+        this.audienceCriteria = 'All non-Firefox users with active adblockers.';
```

Comment:
Why exclude Firefox? The only issue with FF is that FF ABP can't be detected.

We had a means to validate that it was _installed_, but this would return true when the user had whitelisted us, so we made a clear distinction between `adblockInUse` and `adblockerInstalled`. However, the latter no longer exists as it was deemed a potential privacy encroachment, so we just let FF ABP users evade our detection.

Not sure if including FF users would give you better results / a larger sample, but thought it worth mentioning...